images/installer/Dockerfile.ci

```diff
@@ -0,0 +1,13 @@
+# This Dockerfile is a used by CI to publish openshift/origin-v4.0:installer
+
+FROM openshift/origin-release:golang-1.10 AS build
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+RUN hack/build.sh && hack/get-terraform.sh
```

Comment:
We'll have to replace the `curl`-based `get-terraform.sh` here with a `yum` call or some such to comply with Red Hat's OCP Automated Release Policies.  But that's probably not too much of a rush.  I think `get-terraform.sh` should stay `curl`-based to be package-manager agnostic.